lib/modules/home/widgets/story_tile.dart

```diff
@@ -20,7 +20,7 @@ class StoryTile extends StatelessWidget {
         aspectRatio: 1,
         child: Card(
           child: info.imageUrl == null
-              ? const Icon(Icons.landscape_rounded)
+              ? const Icon(Icons.history_edu_rounded)
```

Comment:
What's the reason behind this icon? That a story is like "history" or what? :D

Reply:
Yes, kind of manuscript of a story. My personal association.